### DISCUSSION TITLE
How do I make a custom email address from my domain and connect it to Outlook?

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [vercel/vercel#6564] [Help]
I've already added the MX record via the DNS template, but I don't know where to go from here.

**Comment by barelyhuman** [comment]
You have to configure this on Outlook's personalized email settings, which will then test the MX records for the same and that should basically be it. 

Let me know if you are stuck somewhere else in the process.

**Reply by TheOnlyTails** [reply]
I don't have those settings (I do have premium): 
![image](https://user-images.githubusercontent.com/65342367/128710424-511f61b6-4649-468f-81cb-95ac5843bf6b.png)

**Reply by barelyhuman** [reply]
In that case you're better off approaching people at outlook for the same, they'll guide you through it. 
Currently the docs they have lead to the same page so I'm limited to the same amount of information as you are

**Reply by TheOnlyTails** [reply]
Thanks anyway.
